grapher/Models/Calculations/AccelData.cs

```diff
@@ -82,10 +82,10 @@ public void CalculateDotsXY(int x, int y, double timeInMs)
             EstimatedY.Gain.Set(inYVelocity, yGain);
         }
 
-        public void CalculateDotsCombinedDiffSens(int x, int y, double timeInMs, DriverSettings settings)
+        public void CalculateDotsCombinedDiffSens(double x, double y, double timeInMsRecip, DriverSettings settings)
         {
             (var xStripped, var yStripped) = AccelCalculator.StripSens(x, y, settings.sensitivity.x, settings.sensitivity.y);
-            var outVelocity = AccelCalculator.Velocity(xStripped, yStripped, timeInMs);
+            var outVelocity = AccelCalculator.Magnitude(xStripped, yStripped) * timeInMsRecip;
```

Comment:
Not a huge deal but I'm wondering why this is done instead of just using the helper function I created specifically to do this. Even if this is an optimization, the nice thing about the helper function is that it will allow easier changes in the future (like if we put in the stigma system which specifically would require generalization of that function.)

Reply:
micro-optimizations aside I just thought it was better to not divide by user input

Reply:
sure, can we just add it to the velocity function instead?